docs/source/fuse.rst

```diff
@@ -0,0 +1,34 @@
+.. mdinclude:: ../../fuse/dl/README.md
```

Comment:
(as for your question) lets put it in `fuse.dl.rst`.
Should be automatically once the script will be fixed